src/ImmutableX.ts

```diff
@@ -89,74 +92,131 @@ export class ImmutableX {
   }
 
   /**
-   * Deposits
+   * Deposit based on a token type
+   * @param ethSigner - the L1 signer
+   * @param deposit - the token type amount in its corresponding unit
+   * @returns the transaction
    */
   public deposit(ethSigner: EthSigner, deposit: TokenAmount) {
     return this.workflows.deposit(ethSigner, deposit);
   }
 
+  /**
+   * Get details of a Deposit with the given ID
+   * @param requestParameters - the request object containing the parameters to be provided in the API request
+   * @returns the requested Deposit
+   */
   public getDeposit(requestParameters: DepositsApiGetDepositRequest) {
```

Comment:
@matthewmuscat `requestParameters` -> `request`?

Reply:
I believe the idea is to use `request` for all POST actions (creates/etc) and `requestParameters` for everything else. Although I don't know why we can't just make it `request` for everything?

Reply:
No reason. We can call it `request` in the public interface.

Reply:
https://github.com/immutable/imx-core-sdk/pull/170/commits/c6328988252e1f652abc2ef3afed662a0f907ba4